slack-lint/src/main/java/slack/lint/inclusive/InclusiveNamingChecker.kt

```diff
@@ -25,27 +25,32 @@ import com.android.tools.lint.detector.api.JavaContext
 import com.android.tools.lint.detector.api.Location
 import com.android.tools.lint.detector.api.Position
 import com.android.tools.lint.detector.api.Severity
+import com.android.tools.lint.detector.api.StringOption
 import com.android.tools.lint.detector.api.TextFormat
 import com.android.tools.lint.detector.api.XmlContext
 import org.jetbrains.uast.UElement
 import org.w3c.dom.Node
 import slack.lint.util.Priorities
 import slack.lint.util.resourcesImplementation
 import slack.lint.util.sourceImplementation
-import java.io.StringReader
 import java.util.Locale
-import java.util.Properties
 
 sealed class InclusiveNamingChecker<C : Context, N> {
   companion object {
 
+    internal val BLOCK_LIST =
+      StringOption(
+        "block-list",
+        "A comma-separated list of blocked words from [BLOCK_LIST_PROPERTY] set in the root project's`.",
```

Comment:
`BLOCK_LIST_PROPERTY` doesn't exist anymore.

Reply:
derp, forgot to finish this 3df7eea